index.php

```diff
@@ -47,7 +55,11 @@
 
         <!-- patička -->
         <div class="footer navbar navbar-light bg-light rounded">
-            <p class="mb-0">Spolu s Czechitas jsme to s láskou upekli v LMC s.r.o. | Počet zhlédnutí blogu: <?php echo $pocetZhlednuti; ?></p>
+            <p class="mb-0">
```

Comment:
když includujeme nad hlavičkou nějaká data - nehodil by se i tady include celé patičky?

Reply:
Hele přidal jsem to jako volitelnej úkol na dnešek :upside_down_face: